src/lib/__tests__/unpackPlotProps-test.js

```diff
@@ -0,0 +1,107 @@
+import {applyCustomConfigVisibility} from '../index';
```

Comment:
@nicolaskruchten it would help me if you could think of more example cases to test here..